app/models/post.rb

```diff
@@ -5,6 +5,10 @@ class Post < ApplicationRecord
 
   after_save :update_posts_counter
 
+  validates :title, presence: true, length: { maximum: 250 }
+  validates :comments_counter, comparison: { greater_than: 0 }
```

Comment:
- [ ] CommentsCounter must be an integer greater than or equal to zero, not just greater than zero, kindly implement this so the comment counter can accept also a zero value